Chapter_5/emr/README.md

```diff
@@ -1,55 +1,75 @@
-### Setting up an EMR Cluster for ETL
+### EMR Setup
+ 
+### Prerequisite:
```

Comment:
these headings should have 4 #s to be distinguished from the main heading

Reply:
done